tests/testthat/test-live.R

```diff
@@ -24,38 +24,38 @@ test_that("Scenario Test", {
   }
 
   res
-  
+
   getDoParWorkers()
-  
-  
+
+
   '%dopar%' <- foreach::'%dopar%'
   res <- foreach::foreach(i = 1:4) %dopar% {
     mean(1:i)
   }
-  
+
   res
-  
+
   results <- foreach(i = 1:1) %dopar% {
-    x <- try(library(IRanges),silent=TRUE)
-    x <- try(library(GenomeInfoDb),silent=TRUE)
-    x <- try(library(DESeq2),silent=TRUE)
+    x <- try(library(IRanges), silent = TRUE)
```

Comment:
Separate this test for bioconductor install test?

Reply:
Pablo updated this in container branch, merged.